### PR TITLE
sql: fix multiregion sqlsmith failure

### LINKED ISSUE #78138: sqlsmith: expected 4 destination arguments in Scan, not 2
roachtest.sqlsmith/setup=seed-multi-region/setting=multi-region [failed](https://teamcity.cockroachdb.com/viewLog.html?buildId=4614418&tab=buildLog) with [artifacts](https://teamcity.cockroachdb.com/viewLog.html?buildId=4614418&tab=artifacts#/sqlsmith/setup=seed-multi-region/setting=multi-region) on release-22.1 @ [f5b42a4e3bbdadb25aee89a41871239a624d538d](https://github.com/cockroachdb/cockroach/commits/f5b42a4e3bbdadb25aee89a41871239a624d538d):

```
The test failed on branch=release-22.1, cloud=gce:
test artifacts and logs in: /artifacts/sqlsmith/setup=seed-multi-region/setting=multi-region/run_1
	sqlsmith.go:271,sqlsmith.go:304,test_runner.go:866: Failed generating a query Caught error sql: expected 4 destination arguments in Scan, not 2
```
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

</p>
</details>
/cc @cockroachdb/sql-queries
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*sqlsmith/setup=seed-multi-region/setting=multi-region.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-13987

### PR DESCRIPTION
**Pull request by RichardJCai**
Release note: None

Fixes #78138

### CODE REVIEW

**Review by rytaft — APPROVED** [review]
:lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/84650#-:-N7M3MCDEVxeGvxQ9DoA:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @yuzefovich)

**Review by yuzefovich — APPROVED** [review]
Thanks! :lgtm:

> Reviewed 1 of 1 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/84650#-:-N7ME-LG9BCcaD-z6yIF:b-ftzx47)* status: :shipit: complete! 2 of 0 LGTMs obtained (waiting on @RichardJCai)

___
*[`pkg/internal/sqlsmith/alter.go` line 22 at r1](https://reviewable.io/reviews/cockroachdb/cockroach/84650#-N7MDkpBCCXvtF41qoWy:-N7MDkpCESwmSGPYF_xU:b-5vhppi) ([raw file](https://github.com/cockroachdb/cockroach/blob/e73ef488561e92597e97b1e0d7c12a4712bef583/pkg/internal/sqlsmith/alter.go#L22)):*
> ```Go
> 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree/treebin"
> 	"github.com/cockroachdb/cockroach/pkg/sql/types"
> 	"github.com/cockroachdb/cockroach/pkg/testutils/lint/passes/errwrap/testdata/src/github.com/cockroachdb/errors"
> ```

nit: this could be made shorter.

File: pkg/internal/sqlsmith/alter.go [hunk 1]
```diff
@@ -19,6 +19,7 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree"
 	"github.com/cockroachdb/cockroach/pkg/sql/sem/tree/treebin"
 	"github.com/cockroachdb/cockroach/pkg/sql/types"
+	"github.com/cockroachdb/cockroach/pkg/testutils/lint/passes/errwrap/testdata/src/github.com/cockroachdb/errors"
```
Comment (rafiss) [inline pkg/internal/sqlsmith/alter.go:22]: i think this is the wrong import

**Review by rafiss — CHANGES_REQUESTED** [review]

File: pkg/internal/sqlsmith/alter.go (hunk 1 above)
Comment (RichardJCai) [reply pkg/internal/sqlsmith/alter.go]: Yeah, fixed

**Comment by RichardJCai** [comment]
Thanks for reviewing all!

bors r+

**Pull request merged**